src/platform_impl/android/mod.rs

```diff
@@ -468,15 +469,22 @@ impl EventLoop {
                             &mut self.combining_accent,
                         );
 
+                        let logical_key = keycodes::to_logical(key_char, keycode);
+                        let text = if let Key::Character(ref c) = logical_key {
+                            Some(c.to_owned())
+                        } else {
+                            None
+                        };
```

Comment:
Let's change this to:
```suggestion
                        let text = if state == event::ElementState::Pressed {
                            logical_key.to_text().map(SmolStr::new)
                        } else {
                            None
                        };
```

Which is the abstraction that Windows and AppKit also use to _not only_ convert characters to their textual representation but also keys like `NamedKey::Enter` to `\r`, `NamedKey::Tab` to `\t` andsoforth.


And supposedly this should stay `None` if the key is released, too.